src/coreclr/vm/interpexec.cpp

```diff
@@ -367,6 +382,23 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                     ip += 3;
```

Comment:
Should the non-overflowing conversions above be done the same way? (or the other way around)

Reply:
I'm not sure what you mean. Do you mean doing the non-ovf convs in a helper too? Or are you specifically referring to the floating point ones